### ISSUE TITLE
Jailbreaked Firmware 9.0 / LOGIN_FAILED

### ISSUE THREAD

**Issue by CygnusHyoga** [dhleong/ps4-waker#164]
Hello,

I have been using ps4-waker for a long time... and first of all, I would like to congratulate you on this great project.

I'm having trouble trying to use ps4-waker with a ps4, with jailbreaked firmware 9.0 with the error: "Unexpected error:21 / LOGIN_FAILED". I have read the troubleshooting guide, related to this topic, about problem that you may be logging into a different account on the mobile app than you're using on the PS4 (See #51) but I have tried with the correct account that I have assigned to the ps4 and the ps second screen and I have not been able to solve it.

Credentials are correctly saved in the .ps4-wake.credentials.json.

Could anybode help me? Has anyone been able to use ps4-wake with a jailbroken console (goldhen)? Thanks!

**Comment by CygnusHyoga** [comment]
I had activated a local account with fakeid with PS4OfflineAccountActivator [https://github.com/charlyzard/PS4OfflineAccountActivator](url). I am going to try to activate a real psn account with PS4OfflineAccountActivator (knowing the problems that this entails: I will have to repair saves, trophies, etc.)

**Comment by CygnusHyoga** [comment]
I activate a real psn user_id (hex) and PsnAccountID with PS4OfflineAccountActivator, but without success...I've the same error "Unexpected error: 21 / LOGIN_FAILED". Any help? Thanks!

**Comment by vadimbz** [comment]
Yes please, there's 0 integration options for jailbroken consoles

**Comment by HVR88** [comment]
This repo hasn't been touched in about 2 years unfortunately.

 I guess no one seems very interested in lan wake-up or shut-down/sleep for automation. I can't find anyone else so much as mentioning it in this context. Remote play works on jailbroken systems, and the only time I find anything about wake is with regards to Chiaki - which can do it.

**Comment by tonytwostep** [comment]
Issue still persists. Ended up writing a small Python utility which works on jailbroken / offline-only consoles and can properly wake up both PS4 and PS5 systems from rest mode:
- https://github.com/tonytwostep/playstation-wake-on-lan
